packages/ui/src/composites/wui-balance/styles.ts

```diff
@@ -0,0 +1,16 @@
+import { css } from 'lit'
+
+export default css`
+  span {
+    font-weight: 500;
+    font-size: 40px;
+    color: var(--wui-color-fg-100);
+    line-height: 130%; /* 52px */
+    letter-spacing: -1.6px;
+    text-align: center;
+  }
```

Comment:
isn't this captured by some `TextType` ?

Reply:
This one is really specific for `wui-balance` and won't be needed for other text styles